.github/workflows/erlang.yml

```diff
@@ -10,17 +10,21 @@ jobs:
 
     strategy:
       matrix:
-        otp: [21.3, 22.3, 23.0.4, 23.2.7.0, 24.0]
-    container:
-      image: erlang:${{ matrix.otp }}-alpine
+        combo:
+          - otp: ['21.3', '22.3', '23.3']
```

Comment:
I also wished it was this intelligent, but alas you have to separate all the different combos, it seems. 😢